cpp/GRPCTest.cpp

```diff
@@ -175,6 +175,36 @@ LUA_FUNCTION(UpdateDiscordStatus) {
     LUA->GetField(1, "instance");
     discordP.instance = LUA->GetNumber();
 
+    LUA->GetField(1, "buttonPrimaryLabel");
+    if (LUA->GetType(-1) == GarrysMod::Lua::Type::String) {
+        DiscordRichPresenceButton button;
+
+        button.label = LUA->GetString();
+
+        LUA->GetField(1, "buttonPrimaryUrl");
+        LUA->CheckString();
+
+        button.url = LUA->GetString();
+
+    
```

Comment:
Weird, my VS2022 doesn't show the trailing whitespace at line L189 but when I commit it, it re-appears there

Reply:
All good 👍